### ISSUE TITLE
endless loop, when trying to install software

### ISSUE THREAD

**Issue by edmundlaugasson** [vinifmor/bauh#197]
*.*Describe the bug**
When clicking on Install button, bauh remains endlessly checking any conflicts:
![Screenshot_20210917_171724](https://user-images.githubusercontent.com/6355082/133798142-f777b78e-b449-488f-a271-fd29cdcbc48c.png)
Did also:
```
bauh --reset
sudo bauh --reset
```
... didn't help - same issue repeats...

At the same time at command-line using either pacman or yay, no problem - installs very quickly without issues.

**Software Environment**
bauh version: 0.9.19-1
O.S: EndeavourOS Linux x86_64 (rolling release, all updates done), KDE desktop
Python version: 3.9.7
Installation method: yay + pacman ([AUR](https://aur.archlinux.org/packages/bauh/))

**Comment by vinifmor** [comment]
@edmundlaugasson , launch bauh through the command line using the `--log` parameter and paste the output here, please.
e.g: `bauh --log`

**Comment by edmundlaugasson** [comment]
It was miracle, but today it worked. I started:
`bauh --log 1>bauh.log 2>bauh-error.log`
The file bauh.log remained empty, but bauh-error.log was full of text:
[bauh-error.log](https://github.com/vinifmor/bauh/files/7195495/bauh-error.log)
I did software updates like I usually do. As you see, 3 days ago issue existed. Don't know, how sustainable bauh is, I'm more using command-line (CLI) now. Also not clear, whether bauh deletes downloaded and installed packages or not. At CLI I can give exact command to empty yay, pacman, etc cache.
